crates/rspack_core/src/normal_module.rs

```diff
@@ -364,6 +368,7 @@ impl Module for NormalModule {
   #[tracing::instrument("NormalModule:build", skip_all, fields(
     module.resource = self.resource_resolved_data().resource.as_str(),
     module.identifier = self.identifier().as_str(),
+    id2 = self.resource_data.resource.as_str(),
```

Comment:
Is it possible to make it working agnostic to tracing frameworks? This does not look quite reasonable here.

Reply:
anyway we need two id, one id to bound to same compilation and another id to bound to module resource for module-releated events, so no matter what we tracing framework we choose we do need manually attach special id(it's just called id2 in our case now), I'm open to more general naming